### ISSUE TITLE
Sound volume is lower in 1.98 than with default version 1.93

### ISSUE THREAD

**Issue by DrUm78** [FunKey-Project/picodrive-irixxxx#7]
As spotted by Vincent, the sound volume is louder in default 1.93 version than with the new early build 1.98.
Note that "Sound Volume" setting in config2.cfg has no effect here, tweaking this value does not impact actual sound volume in PicoDrive.

Steps to reproduce:
1. Launch any game at an audible sound volume with 1.98 early build
2. Launch the same game with 1.93 version and at the same sound volume
3. Notice that the sound is louder with default 1.93 build

Expected behavior:
Sound volume should be equal between 1.93 and 1.98 builds.

**Comment by DrUm78** [comment]
@Vincent-FK  I found the way to fix it: https://github.com/DrUm78/picodrive-irixxxx/commit/e9b9b9d81fdfcfeec62fb0b20e72e94344c5bc95 (it's also included in the PR I submitted to your PicoDrive repo).
However it seems that irixxxx did it on purpose, here is his comment:
> Be aware that the full level produces very hearable distortions in some louder games. I reduced the level for a reason. asr 3/>>3 might be a good compromise. OTOH I wouldn't recommend going too high with the shift, or you get kind of a "loudness" effect - lower levels are not reduced as long as the shifting shifts out all of the sound bits.

So far it works for me but I will do some more tests to see if that distortion thing is hearable on the FK speaker.

**Comment by DrUm78** [comment]
Seems like it's a bit louder now, but still lower than 1.93, so I did not fix it entirely. I reopen the issue and will investigate it further.

**Comment by DrUm78** [comment]
Finally found a way to fix it: https://github.com/DrUm78/picodrive-irixxxx/commit/45e3128e13f56f62479533fae9f5550f7848b3bb by deactivating lowpass and DC sound filters that did not work properly anyway on the FK S (it does on RG350P). The logs said:
config_readsect: unhandled var: "Sound filter (low pass)"
config_readsect: unhandled var: "Filter strength (alpha)"

**Comment by DrUm78** [comment]
Deactivating sound filters leads to some sound issues with some 32X games ("Tempo" intro for example). Maybe the best compromise is to keep sound filters activated and just disable the sound volume limiter (which I did in the 1st commit), that's still better than before.

**Comment by DrUm78** [comment]
@Vincent-FK Finally I found the solution:
1. I disabled the sound volume limiter in "mix_arm.S": https://github.com/DrUm78/picodrive-funkey/commit/afad78074279a6b15a69a0052eb5a6731da923dc.
2. The main culprit for the muffled and low sound was the "Sound filter", which is enabled by default. I disabled it by default in the code to prevent this volume issue for all users: https://github.com/DrUm78/picodrive-funkey/commit/6339f601d4317694c80eb4e7e11b2977c8b2156f.

Audio volume is back like in 1.93 version but with all recent fixes (sound in some 32X games like Virtua Fighter or Tempo for example).

Bug fixed and closed.
